manifests/init.pp

```diff
@@ -172,8 +172,8 @@
     manage_repo              => $manage_repo,
   }
```

Comment:
well, no, as you can see, it's instantiated in resource format for reasons not entirely clear to me…